test/fail_compilation/ice13259.d

```diff
@@ -1,8 +0,0 @@
-/*
-TEST_OUTPUT:
----
-fail_compilation/ice13259.d(8): Error: non-constant nested delegate literal expression `__dgliteral3`
----
-*/
-
-auto dg = delegate {};
```

Comment:
Move the test to `compilable`?

Reply:
I should be superseded by the other test I added. But I can if need be.

Reply:
Ehm then just copy paste this line into the other test. I think ICEs are special as we definitely don't want any regressions there.

Reply:
done.